### DISCUSSION TITLE
Search campgrounds with different times

### DISCUSSION THREAD

**Discussion by Dom987911** [juftin/camply#77] [General]
Hello it is possible to search for more campsites with another camp start and end date?

Like this:

camply campsites \
    --rec-area 2725 \
    --start-date 2022-08-01 \
    --end-date 2022-08-03
    --continuous \
    --notifications telegram\
    --search-forever
camply campsites \
    --rec-area 2991\
    --start-date 2022-07-08 \
    --end-date 2022-08-10 \
    --continuous \
    --notifications telegram\
    --search-forever

**Comment by juftin** [comment]
Hey @Dom987911 that functionality isn't currently available but it is on my radar. Here's a link to a more lengthy response on a similar discussion: https://github.com/juftin/camply/discussions/76#discussioncomment-2579106

TLDR;
I hope to build that out soon, and currently the best way to do this is with different docker containers.

**Reply by Dom987911** [reply]
Hey juftin,

thanks for your fast reply, it´s also possible to make a new terminal session with different camply campsites settings?
Ich checked it with the options at the top in seperat terminal session without any problem.
Or is it recommend with docker?

**Reply by juftin** [reply]
Hey Dom. Sorry, I didn't even think of that. Yep, two simultaneous terminal sessions will absolutely work. The end result is just two separate python processes running - but it's completely possible. 

The reason I like docker is because it makes it easy to run camply searches in a container in detached mode in the background. But if the terminal works for you, a new window will definitely work.

**Comment by morenone** [comment]
Hey @juftin , first off, Camply is awesome and it's really helped me and I really appreciate all the work you've done. Since this is a few years old, I'm just checking if this has been implemented? If so, what's the method for docker compose and search.yaml?  If not, how is it accomplished with different containers?  Do you just name them differently? Thanks!

**Comment by juftin** [comment]
Hey @morenone - it's possible to search across [multiple search windows](https://juftin.com/camply/command_line_usage/#searching-across-multiple-time-windows) for the same set of campsites (i.e. campground 123, month of may and month of july) - but not multiple search windows across different campgrounds (i.e. campground 123 in May, 456 in July).

To accomplish this with different containers you would have two different `search.yaml` files mounted into two different containers, but you can have the containers defined in the same `docker-compose.yaml` file. Let me know if that doesn't make sense.

Eventually I'd like to support this feature - just struggling to find the time these days 😅
